api/index.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>DOG Lovers</title>
+    <style>
+        img {
+            width: 200px;
+            height: 200px
+        }
+    </style>
+</head>
+
+<body>
+    <h1>Welcome to my dog website</h1>
+    <h2>Random Dog Image</h2>
+    <img id="randomDogImage" src="" alt="random dog image">
+    <ul id="lists"></ul>
+
+    <!-- another random image -->
+
+    <img id="randomDogImage2" src="" alt="something">
```

Comment:
If you are not going to provide the `alt` attribute with something meaningful, then I'd rather leave it out. I understand it's difficult here because the image changes every 2 seconds, and the message would need to be adjusted if you want to be precise about the race that's being shown. But I'd either try to be general but meaningful (e.g. `"an image of a dog from a random breed"`) or find a solution to make it tailored to the breed (e.g. split the file name/location so you end up with the name of the breed, and then create the alt-text like that.)